src/validation/nationality.validation.ts

```diff
@@ -27,20 +27,23 @@ export const nationalityValidator = async (req: Request, res: Response, next: Ne
     const nationality1 = getField(req, DirectorField.NATIONALITY_1);
     const nationality2 = getField(req, DirectorField.NATIONALITY_2);
     const nationality3 = getField(req, DirectorField.NATIONALITY_3);
-    const isUpdate = req.path.includes("update-director-name");
+    const isUpdate = req.path.includes("update-director-name") || req.path.includes("update-director-nationality");
```

Comment:
Should this be just checking for nationality? and was wrongly checking for name so should check no longer?

Reply:
yep, think this was an existing error - have removed the name section